funcframework/framework.go

```diff
@@ -96,11 +97,49 @@ func RegisterCloudEventFunctionContext(ctx context.Context, path string, fn func
 	return registerCloudEventFunction(ctx, path, fn, handler)
 }
 
+// Declaratively registers a HTTP function.
+func HTTP(name string, fn func(http.ResponseWriter, *http.Request)) {
+	defer recoverPanic("Registration panic")
```

Comment:
This and down below should be removed now that RegisterHTTP has been changed not to panic.

Reply:
Makes sense, I've removed this now.